targets/XPlatCppSdk/source/code/include/playfab/PlayFabJsonHeaders.h

```diff
@@ -1,21 +1,21 @@
 #pragma once
 
-#ifdef _DURANGO
+#include <playfab/PlayFabPlatformMacros.h>
+
+#ifdef PLAYFAB_PLATFORM_XBOX
 #include <json/json.h>
 #include <json/reader.h>
 #include <json/value.h>
```

Comment:
It looks like these are the same for Xbox and Windows. Can we possibly unite those with one condition defined(xbox) || defined (windows)?